CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(OPEN62541_CPP open62541cpp)
 ## build settings - try and match the c library version
 set(VERSION_MAJOR 1)
 set(VERSION_MINOR 2)
-set(VERSION_PATCH 666)
+set(VERSION_PATCH 777)
```

Comment:
autocrane_core has to be updated to fetch this version

Reply:
and it should be tagged once this pr is merged

Reply:
tagged as v1.2.77 ->  https://github.com/psiori/open62541Cpp/releases/tag/v1.2.777